src/SConscript

```diff
@@ -618,35 +618,35 @@ def win32_find_program_via_registry(program_name):
 
 def BuildRelease(target, source, env):
     print
-    print "==== Mixxx Post-Build Checks ===="
+    print("==== Mixxx Post-Build Checks ====")
     print
-    print "You have built version %s" % mixxx_version
+    print("You have built version %s" % mixxx_version)
     print
     if build.build_is_debug:
-        print "YOU ARE ABOUT TO PACKAGE A DEBUG BUILD!!"
+        print("YOU ARE ABOUT TO PACKAGE A DEBUG BUILD!!")
         print
-    print "Binary has size ",
+    print("Binary has size ")
```

Comment:
extra carriage return here.
Expected result on screen is
```
Binary has size 31888384
```

Actual result is 
```
Binary has size 
31888384
```

see line 11485 to 11492 of https://ci.appveyor.com/project/mixxxdj/mixxx/build/master-2608/job/buaxyoi627i7uud8

Use this form instead:
```python
print("Binary has size ", end='')
```

Reply:
for this one, you can't automate it when comma at the end of line because of https://github.com/mixxxdj/mixxx/blob/master/src/SConscript#L1021